addons/gamification/data/goal_base.xml

```diff
@@ -145,7 +145,7 @@ Thank you,
                                                                 <div style="height:${heightA}px;">
                                                                     ${extra_div | safe}   
                                                                     <div style="height:55px;">
-                                                                        <img style="margin-bottom:5px;width:50px;height:50px;border-radius:50%;" t-att-src="image_data_uri(object.env['res.users'].browse(goal['user_id']).partner_id.image_64)" alt="${goal['name']}"/>
+                                                                        <img style="margin-bottom:5px;width:50px;height:50px;border-radius:50%;object-fit:cover;" t-att-src="image_data_uri(object.env['res.users'].browse(goal['user_id']).partner_id.image_128)" alt="${goal['name']}"/>
```

Comment:
Why not the class ?

Reply:
Mail template so class not available there.

Reply:
Not sure how email template work now but I think there is a "clean" version with classes saved in db and a "transcoded" version with classes *and* inline styles saved in db (while it was only that one in 12.0)... so in any case, the classes should always be there I think

Reply:
I checked again to be sure, but I don't get any CSS on the mails that are sent.

Reply:
There is never css but the classes should be there